### ISSUE TITLE
Possible Bug

### ISSUE THREAD

**Issue by SteelBlood2** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#219]
Are perks that add extra ammo supposed to apply to crossbows? Because so far none of them do. Be a bit weird to have perks adding ammo to throwing and bows but not crossbows.

**Comment by PaddyVu** [comment]
> Are perks that add extra ammo supposed to apply to crossbows? Because so far none of them do. Be a bit weird to have perks adding ammo to throwing and bows but not crossbows.

yeah i do notice it,i think crossbow still overpower because headshot will instant kill anything at close range,with crossbow perk i can reload on horse,pretty op right?

**Comment by Tyler-IN** [comment]
We might have a PR that addresses this, not sure...

**Comment by fnzr** [comment]
Extra ammo perks intentionally don't apply to crossbows. In the Riding and Athletics tree, perks are explicitly named "Spare Arrows" and "Extra Arrows" and add +X arrows, not bolts.

**Comment by Tyler-IN** [comment]
Is this good to close?
